indexing-hadoop/src/test/resources/druid.test.data.tsv

```diff
@@ -0,0 +1,82 @@
+2011-04-10T00:00:00.000Z	spot	automotive	preferred	apreferred	113.221448
```

Comment:
is there a way we can commonalize the test files to avoid copy and pasting them all over the place?